app/config/FrontendAppConfig.scala

```diff
@@ -40,7 +40,7 @@ class FrontendAppConfig @Inject() (configuration: Configuration) {
   val signOutContinueUrl: String = configuration.get[String]("urls.signOutContinue")
 
   private val exitSurveyBaseUrl: String = configuration.get[Service]("microservice.services.feedback-frontend").baseUrl
-  val exitSurveyUrl: String             = s"$exitSurveyBaseUrl/feedback/trader-goods-profiles-frontend"
+  val exitSurveyUrl: String             = s"$exitSurveyBaseUrl/feedback-form/trader-goods-profiles-frontend"
```

Comment:
Should just be /feedback/<service-name> so what we had before was fine

https://github.com/hmrc/feedback-frontend?tab=readme-ov-file#service-manager